verifier-sdk/Cargo.toml

```diff
@@ -19,8 +24,12 @@ ark-ff = { version = "^0.3.0", default-features = false }
 ark-ec = { version = "0.3.0" }
 ark-bn254 = "0.3.0"
 ark-std = { version = "^0.3.0", default-features = false }
-spl-token = "3.3.0"
 borsh = "0.9.3"
 groth16-solana = "0.0.2"
+light-macros = { version = "0.3.1", path = "../macros/light" }
+spl-token = "3.3.0"
 light-utils = { path = "../utils", version = "0.1.0" }
 light-sparse-merkle-tree = { path = "../merkle-tree/sparse", version = "0.1.1" }
+
+[build]
+target = "x86_64-unknown-linux-gnu"
```

Comment:
Is it really needed? Do you receive any error when removing that line? I think it might potentially break `cargo build` on:

* macOS
* Linux on arm64
* Even on x86_64, it might break Linux distros which are using [musl libc](https://www.musl-libc.org/) instead of [GNU libc](https://www.gnu.org/software/libc/) as the standard C library - for example, [Alpine](https://www.alpinelinux.org/). I'm using musl as well in the most of my setups. :sweat_smile: Such distros usually use `x86_64-unknown-linux-musl` as a target.

Reply:
No it’s not needed